velox/core/PlanNode.cpp

```diff
@@ -443,6 +443,109 @@ PlanNodePtr AggregationNode::create(const folly::dynamic& obj, void* context) {
       deserializeSingleSource(obj, context));
 }
 
+namespace {
+RowTypePtr getExpandOutputType(
+    const std::vector<std::vector<TypedExprPtr>>& projections,
+    std::vector<std::string>& names) {
```

Comment:
pass names as const &

Reply:
@mbasmanova The names is used to construct the Row type by std::move [here](https://github.com/facebookincubator/velox/pull/5403/files#diff-53acad6c52445d1077c935fd55250184d5aecb3e0e4a74ace2b90a51d83de0e1R462). There will be compile error if using const.